[ADD] stock_picking_invoice_link: stock picking reports as attachment…

catalin-domatix opened the pull request:
Given an invoice with pickings, add its pickings `Picking Operations` pdf reports as attachments to the email created by the `Send by Email` invoice button.

catalin-domatix committed:
[ADD] stock_picking_invoice_link: stock picking reports as attachments to the email created by the `Send by Email` button`

pedrobaeza commented:
This is something that shouldn't be done as a general thing, as not everyone wants this. I would suggest an option at res.partner level, company level or system-wide parameter for deciding this.

catalin-domatix committed:
[ADD] stock_picking_invoice_link: settings option for the picking reports as attachments to the to-be-sent  email option; this is:
the group_account_manager users will have to manually select this option if picking reports as atstachments to the email are desired

catalin-domatix committed:
[FIX] stock_picking_invoice_link: missing res_config_settings file

catalin-domatix committed:
[ADD] stock_picking_invoice_link: unittests to evaluate the attachment of picking reports to the invoice email template

catalin-domatix commented:
> Hey @catalin-domatix, thank you for your Pull Request.
> 
> It looks like some users haven't signed our **C**ontributor **L**icense **A**greement, yet.
> You can read and sign our full Contributor License Agreement here: http://odoo-community.org/page/cla
> Here is a list of the users:
> 
>     * @catalin-domatix (login unknown in OCA database)
> 
> 
> Appreciation of efforts,
> OCA CLAbot

From what my colleagues told me this issue has been resolved. 

On the other hand, the Travis lint checkers have failed on files which are alien to the files modified on this PR, so having added an option for the user to select if they want to have the picking reports attached to the email, I'd ask if there is something more to do in order for this PR to be accepted.

rousseldenis commented:
@catalin-domatix Could you please fix lint errors (flake8) ?

catalin-domatix committed:
[UPD] stock_picking_invoice_link: spanish internationalization for new picking reports as email attachments and some codestyle fixes

rousseldenis reviewed (approved):
Code review.

rousseldenis commented:
@catalin-domatix Could you rebase ?

catalin-domatix committed:
[UPD] Updated stock_picking_invoice_link's account_invoice in order to rebase. Added dot pdf to the picking attachment filename

catalin-domatix committed:
[FIX] stock_picking_invoice_link: fixed bad merge caused by the last rebase (see c7190e20820615ceb8c9476f69ba675692dae061 test) and consequent flake and lint errors

rousseldenis commented:
@catalin-domatix Wrong rebase. Could you fix it ?

catalin-domatix commented:
@rousseldenis what exactly is wrong with the rebase tho? i genuinely don't know

rousseldenis commented:
@catalin-domatix Ok, don't see anymore. Seems to be my browser cache. :

rousseldenis commented:
Are all your commits relevant or do you want to squash some of them ?

catalin-domatix commented:
i'd say none of them are relevant besides the first and second one but squashing is not trivial since i rebased, there are commits which are not mine in between my own ones, gimme a minute

pedrobaeza commented on stock_picking_invoice_link/i18n/es.po:
```diff
@@ -101,5 +101,22 @@ msgstr "Transferir"
 msgid "You can not modify an invoiced stock move"
 msgstr "No puede modificar un movimiento de almacén facturado"
 
+<<<<<<< HEAD
```
This conflict must be solved

rousseldenis replied:
That's what I wanted to say. My memory is limited :smile:

pedrobaeza commented on stock_picking_invoice_link/readme/CONTRIBUTORS.rst:
```diff
@@ -18,7 +18,9 @@
   * David Vidal <david.vidal@tecnativa.com>
+* `Domatix <https://www.domatix.com>`_
 
+  * Catalin Airimitoaie <catalin@domaitx.com>
```
Incorrect email address

pedrobaeza commented on stock_picking_invoice_link/readme/DESCRIPTION.rst:
```diff
@@ -1,13 +1,18 @@
 This module adds a link between pickings and invoices as well as on the lines.
-Invoices are generated from sales orders. With this module, you can find back
+Invoices are generated from sales orders. With this module you can find back
```
Why do yo remove this valid comma?
